docs/content/en/docs/getting-started/optional/cni.md

```diff
@@ -168,7 +170,11 @@ spec:
 
 The [Cilium CLI](https://github.com/cilium/cilium-cli) can be used to uninstall EKS Anywhere Cilium 
 via `cilium uninstall`.
-See the [replacing Cilium task]({{< ref "/docs/tasks/cluster/cluster-replace-cilium" >}}) for a walkthrough on how to successfully replace EKS Anywhere Cilium.
+<<<<<<< HEAD
+See the [replacing Cilium task]({{< ref "/docs/clustermgmt/networking/cluster-replace-cilium" >}}) for a walkthrough on how to successfully replace EKS Anywhere Cilium.
+=======
```

Comment:
^ Same here. Leftover rebase code

Reply:
Ah, another one. Thanks for finding. It'll be in the next commit.